### PR TITLE
[lint-tool] Fix lint errors for non-upstream test files in webapi

### PR DESCRIPTION
**Pull request by Honry**
https://crosswalk-project.org/jira/browse/CTS-1438

### CODE REVIEW

**Comment by Honry** [comment]
@haoyunfeix @haoxli , PTAL.

File: webapi/tct-animationtiming-w3c-tests/animationtiming/AnimationTiming_Callback_FrameRequestCallback_multipleCalls.html [hunk 1]
```diff
@@ -36,6 +36,7 @@
     <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
     <title>Multiple callback registrations occur</title>
     <script type="text/javascript" src="../resources/testharness.js"></script>
+    <script type="text/javascript" src="../resources/testharnessreport.js"></script>
     <link rel="stylesheet" href="../resources/testharness.css" />
```
Comment (haoxli) [inline webapi/tct-animationtiming-w3c-tests/animationtiming/AnimationTiming_Callback_FrameRequestCallback_multipleCalls.html]: I think testharness.css can  be removed.

File: webapi/tct-animationtiming-w3c-tests/animationtiming/AnimationTiming_Callback_FrameRequestCallback_validTimestamp.html [hunk 2]
```diff
@@ -40,6 +40,7 @@
     <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
     <title>Check whether FrameRequestCallback contains a valid timestamp</title>
     <script type="text/javascript" src="../resources/testharness.js"></script>
+    <script type="text/javascript" src="../resources/testharnessreport.js"></script>
     <link rel="stylesheet" href="../resources/testharness.css" />
```
Comment (haoxli) [inline webapi/tct-animationtiming-w3c-tests/animationtiming/AnimationTiming_Callback_FrameRequestCallback_validTimestamp.html]: I think testharness.css can  be removed.

File: webapi/tct-video-html5-tests/video/video_zero_default_state.html [hunk 3]
```diff
@@ -39,14 +39,14 @@
...
+    <script src="../common/media.js"></script>
   </head>
   <body>
     <div id="log"></div>
     <video id="m">Your browser does not support media elements.</video>
     <script>
-		    var media = document.getElementById("m");
-		    media.src = getVideoURI('media/movie_5');
+            var media = document.getElementById("m");
+            media.src = getVideoURI('media/movie_5');
```
Comment (haoxli) [inline webapi/tct-video-html5-tests/video/video_zero_default_state.html]: Reduce four more space

File: webapi/tct-animationtiming-w3c-tests/animationtiming/AnimationTiming_Callback_FrameRequestCallback_multipleCalls.html (hunk 1 above)
Comment (Honry) [inline webapi/tct-animationtiming-w3c-tests/animationtiming/AnimationTiming_Callback_FrameRequestCallback_multipleCalls.html]: That can be another lint error.

**Comment by haoyunfeix** [comment]
lgtm

**Pull request merged**